packages/flutter_tools/test/integration.shard/flutter_attach_test.dart

```diff
@@ -197,24 +197,22 @@ void main() {
     testWithoutContext('enables Observatory on run', () async {
         await flutterRun.run(
           withDebugger: true,
-          // TODO(bkonyi): uncomment once Observatory is disabled by default
-          // See https://github.com/dart-lang/sdk/issues/50233
-          // serveObservatory: true,
+          serveObservatory: true,
         );
         expect(await isObservatoryAvailable(), true);
     });
 
     testWithoutContext('enables Observatory on attach', () async {
```

Comment:
Should this test have a new name?

Reply:
No, this test was named this way in anticipation of Observatory being disabled by default.